osf/models/subject.py

```diff
@@ -1,25 +1,33 @@
 # -*- coding: utf-8 -*-
 from django.db import models
+from django.db.models import Q
 from django.core.exceptions import ValidationError
 from django.utils.functional import cached_property
+from include import IncludeQuerySet
 
 from website.util import api_v2_url
 
-from osf.models.base import BaseModel, ObjectIDMixin
+from osf.models.base import BaseModel, MODMCompatibilityQuerySet, ObjectIDMixin
 from osf.models.preprint_provider import PreprintProvider
 from osf.models.validators import validate_subject_hierarchy_length, validate_subject_provider_mapping
 
+class SubjectQuerySet(MODMCompatibilityQuerySet, IncludeQuerySet):
```

Comment:
Does this need to inherit from IncludeQuerySet? I don't see any usages of .include anywhere.